sdk/storage/azdatalake/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+* Escape paths for NewDirectoryClient and New FileClient in a file system
```

Comment:
You can mention the bug here, like
```suggestion
* Escape paths for NewDirectoryClient and New FileClient in a file system. Fixes [#22281](https://github.com/Azure/azure-sdk-for-go/issues/22281).
```

Reply:
*  [#22281](https://github.com/Azure/azure-sdk-for-go/issues/22281): Escape paths for NewDirectoryClient and New FileClient in a file system

Reply:
mentioned the bug in changelog, thanks!